src/components/rules/menu.tsx

```diff
@@ -77,32 +77,26 @@ export const RuleSetMenu = ({ id }: { id: string }) => {
 
 export const NewRuleSetMenu = ({ id }: { id: string }) => {
   return (
-    <Navbar fluid rounded>
```

Comment:
Wrapping the dropdown in a navbar was causing errors in the browser console about invalid <li> nesting.